doc/source/RSE_Expressions.rst

```diff
@@ -49,11 +48,21 @@ Rucio allows to test RSE Expressions, using the command **list-rses**. The most
     IFIC-LCG2_DATADISK
     LIP-COIMBRA_LOCALGROUPDISK
 
+5) Also numerical comparisons with < and > are possible::
 
-Note that if the RSE Expresion returns an empty set, rucio returns an error. This could be because the name of the attribute doesn't exist or because there's no RSE that match the expression. It does not necessarily mean that the syntax of the expression is wrong.
+     jbogadog@lxplus0058:~$ rucio list-rses --expression "freespace>3000"
+     CERN-PROD_TZDISK
+     BNL-OSG2_MCTAPE
+     BNL-OSG2_DATADISK
+     IN2P3-CC_MCTAPE
+     CERN-PROD_DERIVED
+     CERN-PROD_DATADISK
+
+Note that if the RSE Expresion returns an empty set, rucio returns an error as an RSE Expression must resolve to at least one RSE. Thus, an error does not necessarily mean that the syntax of the expression is wrong, it might just resulted into an empty list.
 
 In 2) and 3), the RSE Expression refers to an attribute in the RSE that must be equal to a given value to match the expression.
-While in 1) and 4), the expression match a RSE if the attribute is True. It is possible to see the list of attributes for a particular RSE with rucio::
+While in 1) and 4), the expression match a RSE if the attribute is True. In 5) a numerical term is used to resolve all RSEs with more than 3000 TB free space.
+It is possible to see the list of attributes for a particular RSE with rucio::
```

Comment:
hmm, ah wait, you're actually refering to rucio the CLI. It can be in this backtick-quote then? Otherwise I would suggest to keep Rucio always capital.